quarkus-bon-jova-rockstar/deployment/src/main/resources/dev-ui/qwc-bon-jova-rockstar-endpoints.js

```diff
@@ -5,14 +5,16 @@ import '@vaadin/grid';
 import '@vaadin/grid/vaadin-grid-sort-column.js';
 import '@vaadin/details';
 import 'echarts-gauge-grade';
-import 'qui-code-block';
+import '@quarkus-webcomponents/codeblock';
 import 'qui-ide-link';
 import 'qui-badge';
+import {observeState} from "./lit-state";
+import {themeState} from "./theme-state";
```

Comment:
This should be :

```
import { observeState } from 'lit-element-state';
import { themeState } from 'theme-state';
```

I'll update the documentations to include this.  All that this does is adopt the correct codemirror theme based on the dev-ui theme.